stripes-web/src/test/java/org/stripesframework/web/controller/BindingSecurityTests.java

```diff
@@ -223,23 +265,126 @@ public void setFoo( String foo ) {
 
-   @StrictBinding(allow = "foo,bar")
-   public static class ImplicitDeny extends BindingSecurityTests.NoAnnotation {
+   @SuppressWarnings("unused")
+   public static class NestedListValidateAnnotations extends BindingSecurityTests.NoAnnotation {
+
+      private final List<Blah> _blahList = new ArrayList<>();
+
+      @ValidateNestedProperties({ //
+                                  @Validate(field = "name"), //
+                                })
+      public List<Blah> getBlahList() {
+         return _blahList;
+      }
 
       @Override
       public boolean[] getExpectSuccess() {
-         return new boolean[] { true, true, false };
+         return new boolean[] { true, false };
       }
+
+      @Override
+      public String[] getTestProperties() {
+         return new String[] { "blahList[0].name", "blahList[0].internalName" };
+      }
+
+   }
+
+
+   @SuppressWarnings("unused")
+   public static class NestedListValidateAnnotationsDotNotation extends BindingSecurityTests.NoAnnotation {
+
+      private final List<Blah> _blahList = new ArrayList<>();
+
+      @ValidateNestedProperties({ //
+                                  @Validate(field = "name"), //
+                                })
+      public List<Blah> getBlahList() {
+         return _blahList;
+      }
+
+      @Override
+      public boolean[] getExpectSuccess() {
+         return new boolean[] { true, false };
+      }
+
+      @Override
+      public String[] getTestProperties() {
+         return new String[] { "blahList.0.name", "blahList.0.internalName" };
+      }
+
+   }
+
+
+   @SuppressWarnings("unused")
+   public static class NestedMapValidateAnnotations extends BindingSecurityTests.NoAnnotation {
+
+      private final Map<String, Blah> _blahMap = new HashMap<>();
+
+      @ValidateNestedProperties({ //
+                                  @Validate(field = "name"), //
+                                })
+      public Map<String, Blah> getBlahMap() {
+         return _blahMap;
+      }
+
+      @Override
+      public boolean[] getExpectSuccess() {
+         return new boolean[] { true, false };
+      }
+
+      @Override
+      public String[] getTestProperties() {
+         return new String[] { "blahMap[foo].name", "blahMap[foo].internalName" };
+      }
+
+   }
+
+
+   @SuppressWarnings("unused")
+   public static class NestedMapValidateAnnotationsDotNotation extends BindingSecurityTests.NoAnnotation {
+
+      private final Map<String, Blah> _blahMap = new HashMap<>();
+
+      @ValidateNestedProperties({ //
+                                  @Validate(field = "name"), //
+                                })
+      public Map<String, Blah> getBlahMap() {
+         return _blahMap;
+      }
+
+      @Override
+      public boolean[] getExpectSuccess() {
+         return new boolean[] { true, false };
+      }
+
+      @Override
+      public String[] getTestProperties() {
+         return new String[] { "blahMap.foo.name", "blahMap.foo.internalName" };
```

Comment:
Binding of `blahMap.foo.name` should throw an `EvaluationException` instead of being bound, because it needs to be `blahMap[foo].name`.

Reply:
The test does exactly that.